librad/src/keys.rs

```diff
@@ -156,6 +175,27 @@ impl SecretKeyExt for SecretKey {
     fn metadata(&self) -> Self::Metadata {}
 }
 
+#[async_trait]
+impl sign::Signer for SecretKey {
```

Comment:
The `keystore` crate already provides an implementation of `Signer` for `sodiumoxide` keys albeit a bit differently. We should either remove that implementation or use it here.

Reply:
I'm still going to have to `impl sign::Signer for SecretKey` because of the newtype here. Is this to ensure that we use the `Signer` implementation of the `sodiumoxide` keys?

Reply:
You’re right, we need to implement `Signer` anyways. But ideally for the implementation we would just delegate to what `keystore` gives. This would require us to adjust the implementation in `keystore`. I think we can leave this for some other time.

Reply:
I did it anyways :) https://github.com/radicle-dev/radicle-keystore/pull/4